tests/python_tests/ov_genai_test_utils.py

```diff
@@ -223,4 +222,5 @@ def load_pipe(configs: List[Tuple], temp_path):
 
 @functools.lru_cache(1)
 def get_continuous_batching(path):
-    return ov_genai.LLMPipeline(str(path), ov_genai.Tokenizer(str(path)), 'CB')
+    default_scheduler_config = ov_genai.SchedulerConfig
```

Comment:
SchedulerConfig is a class, not an object
Looks like you need to create instance of this class here.

Reply:
done